messaginginapp/build.gradle

```diff
@@ -49,13 +52,17 @@ dependencies {
     api project(":base")
     api project(":sdk")
 
+    implementation 'androidx.core:core-ktx:1.10.1'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.6.1"
+
+    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
```

Comment:
I didn't put them there, because in the refactor they would go away. Instead of `retrofit2:converter-gson:2.9.0'` we will use the `moshi` one we are already using to reduce number of dependencies etc.